### ISSUE TITLE
can not change background and "choice" and normal style is mixed

### ISSUE THREAD

**Issue by pigzerowan** [JoseBarreto1/chips_choice_null_safety#3]

**Comment by pigzerowan** [comment]
cannot change background  and  choce is miexed wiith normal style

**Comment by slavap** [comment]
@JoseBarreto1 
All color code in C2Chip is wrong (isDark may use hardcoded colors, but definitely not take them chaotically from effectiveStyle). Also borderColor from effectiveStyle is not used at all, which is wrong.
```dart
final bool isDark = effectiveStyle.brightness == Brightness.dark;

    final Color? textColor = isDark
      ? const Color(0xFFFFFFFF)
      : effectiveStyle.color;

    final Color? borderColor = isDark
      ? const Color(0x00000000)
      : textColor!.withOpacity(effectiveStyle.borderOpacity ?? defaultBorderOpacity);

    final Color? checkmarkColor = isDark
      ? textColor
      : activeStyle.color;

    final Color? backgroundColor = isDark
      ? style.color
      : const Color(0x00000000);

    final Color? selectedBackgroundColor = isDark
      ? activeStyle.color
      : const Color(0x00000000);
```

**Comment by jbarretojr** [comment]
resolved in 2.0.5

**Comment by pigzerowan** [comment]
> resolved in 2.0.5

can I get your contact infomation?
